hack/generator/pkg/astmodel/function.go

```diff
@@ -11,14 +11,17 @@ import (
 
 // Function represents something that is an (unnamed) Go function
 type Function interface {
+	// The unique name of this interface
```

Comment:
```suggestion
	// The name of this function
```

Reply:
Could keep "unique" too although it's sorta wonky because the next question is "unique in what context" and that depends on where the function is used.

Reply:
Fixed to refer to `function` instead of `interface`. Thanks for catching the typo.
Also extended the comment to explain how the name needs to be unique.